hack/upstream-api/local-setup-mgc-api.sh

```diff
@@ -104,6 +113,9 @@ if [[ -n "${MGC_WORKLOAD_CLUSTERS_COUNT}" ]]; then
     deployOCMSpoke ${KIND_CLUSTER_WORKLOAD}-${i}
     deployPrometheusForFederation ${KIND_CLUSTER_WORKLOAD}-${i}
     configureManagedAddon ${KIND_CLUSTER_CONTROL_PLANE} ${KIND_CLUSTER_WORKLOAD}-${i}
+    deployGatekeeper ${KIND_CLUSTER_WORKLOAD}-${i}
+    configureGatekeeper ${KIND_CLUSTER_WORKLOAD}-${i}
+    installGatekeeperScoreCard ${KIND_CLUSTER_WORKLOAD}-${i}
```

Comment:
This is fine to get things installed with examples in all clusters.
We'll definitely end up changing things for the quickstart as there's 1 hub cluster and 2 spokes.
Some constraints belong only on the hub, while others are for the app developer and belong in the workload clusters.

Reply:
i keep thinking of all hubs being both hub and spoke rather then a hub just being a hub. ill update the script to be able to specify where the constraints should go i don't think it makes sense for the make target to be different to the quickstart.

Reply:
ok grand

Reply:
@david-martin I don't think we showed any for an app dev, only PE ones (at the "hub")